src/translator/service.h

```diff
@@ -38,18 +65,21 @@ class Service {
 
 public:
   /// @param options Marian options object
-  /// @param model_memory byte array (aligned to 64!!!) that contains the bytes
+  /// @param modelMemory byte array (aligned to 256!!!) that contains the bytes
   /// of a model.bin. Optional, defaults to nullptr when not used
-  explicit Service(Ptr<Options> options, const void *model_memory = nullptr);
+  /// @param shortlistMemory byte array of shortlist
+  explicit Service(Ptr<Options> options, AlignedMemory modelMemory, AlignedMemory shortlistMemory);
+
+  explicit Service(Ptr<Options> options) : Service(options, AlignedMemory(), AlignedMemory()){}
```

Comment:
A slight worry I have here is that it's not straighforward what `AlignmentMemory()` is from reading (I'm guessing this allocates memory/set `nullptr` and set's size to 0?). Since we have our own `alignment.h` now, would it be preferable to have a [named constructor](https://isocpp.org/wiki/faq/ctors#named-ctor-idiom) - `AlignedMemory::EmptyMemory()` or something?

Reply:
No.  Default constructors are fine.